lib/residence/residence_screen.dart

```diff
@@ -0,0 +1,564 @@
+                                      ],
+                                    ),
+                                  ),
+                                  Row(
+                                    children: const [
+                                      Padding(
+                                        padding: EdgeInsets.only(right: 5.0),
+                                        child: Icon(
+                                          Icons.train,
+                                          size: 16,
+                                        ),
+                                      ),
+                                      Text(
+                                        '京急本線 京急川崎駅 より 徒歩9分',
+                                        style: TextStyle(fontSize: 11),
+                                      ),
+                                    ],
+                                  ),
+                                  Row(
+                                    children: const [
+                                      Padding(
+                                        padding: EdgeInsets.only(right: 5.0),
+                                        child: Icon(
+                                          Icons.house,
+                                          size: 16,
+                                        ),
+                                      ),
+                                      Text(
+                                        '1K / 21.24㎡ 南西向き',
+                                        style: TextStyle(fontSize: 11),
+                                      ),
+                                    ],
+                                  ),
+                                  Row(
+                                    children: const [
+                                      Padding(
+                                        padding: EdgeInsets.only(right: 5.0),
+                                        child: Icon(
+                                          Icons.apartment,
+                                          size: 16,
+                                        ),
+                                      ),
+                                      Text(
+                                        '2階/15年建 築5年',
+                                        style: TextStyle(fontSize: 11),
+                                      ),
+                                    ],
+                                  ),
+                                ],
+                              ),
+                            ),
+                          ),
+                          Row(
+                            children: [
+                              Padding(
+                                padding: const EdgeInsets.all(10.0),
+                                child: SizedBox(
+                                  width: 170,
+                                  child: Container(
+                                    decoration: BoxDecoration(
```

Comment:
ここはSizedBoxで指定しているwidthは子のContainerで一緒に指定してしまっていいかなと思いました！
```suggestion
                                  child: Container(
                                    width: 170,
                                    decoration: BoxDecoration(
```

Reply:
なるほど...sizedboxいらないのか...ありがとうございます！